### PR TITLE
DOC-5851: Remove unnecessary redirects.

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-5851

### CODE REVIEW

File: _includes/v21.1/sql/movr-statements-geo-partitioned-replicas.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 The following examples use MovR, a fictional vehicle-sharing application, to demonstrate CockroachDB SQL statements. For more information about the MovR example application and dataset, see [MovR: A Global Vehicle-sharing App](movr.html).
 
-To follow along, run [`cockroach demo`](cockroach-demo.html) with the `--geo-partitioned-replicas` flag. This command opens an interactive SQL shell to a temporary, 9-node in-memory cluster with the [Geo-Partitioned Replicas Topology](topology-geo-partitioned-replicas.html) applied to the `movr` database.
+To follow along, run [`cockroach demo`](cockroach-demo.html) with the `--geo-partitioned-replicas` flag. This command opens an interactive SQL shell to a temporary, 9-node in-memory cluster with the `movr` database.
```
Comment (rmloveland) [inline _includes/v21.1/sql/movr-statements-geo-partitioned-replicas.md:5]: Interesting - was this a redirect?  I looked at `v21.1/topology-geo-partitioned-replicas.md` and it looks like a normal docs page

File: _includes/v21.2/sql/movr-statements-geo-partitioned-replicas.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 The following examples use MovR, a fictional vehicle-sharing application, to demonstrate CockroachDB SQL statements. For more information about the MovR example application and dataset, see [MovR: A Global Vehicle-sharing App](movr.html).
 
-To follow along, run [`cockroach demo`](cockroach-demo.html) with the `--geo-partitioned-replicas` flag. This command opens an interactive SQL shell to a temporary, 9-node in-memory cluster with the [Geo-Partitioned Replicas Topology](../v21.2/topology-geo-partitioned-replicas.html) applied to the `movr` database.
+To follow along, run [`cockroach demo`](cockroach-demo.html) with the `--geo-partitioned-replicas` flag. This command opens an interactive SQL shell to a temporary, 9-node in-memory cluster with the `movr` database.
```
Comment (rmloveland) [inline _includes/v21.2/sql/movr-statements-geo-partitioned-replicas.md:5]: Same question about this one, looks like the page exists and is not a redirect - curious why we are removing links in this PR?

I'm sure I'm missing something, let me know if so tho!

File: _includes/v21.2/sql/replication-zone-patterns-to-multiregion-sql-mapping.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
-| [Duplicate indexes](topology-duplicate-indexes.html)                  | [`GLOBAL` tables](global-tables.html)                                                                                                                       |
-| [Geo-partitioned replicas](topology-geo-partitioned-replicas.html)        | [`REGIONAL BY ROW` tables](regional-tables.html#regional-by-row-tables) with [`ZONE` survival goals](multiregion-overview.html#surviving-zone-failures)     |
-| [Geo-partitioned leaseholders](topology-geo-partitioned-leaseholders.html) | [`REGIONAL BY ROW` tables](regional-tables.html#regional-by-row-tables) with [`REGION` survival goals](multiregion-overview.html#surviving-region-failures) |
+| [Duplicate indexes](../v20.2/topology-duplicate-indexes.html)                  | [`GLOBAL` tables](global-tables.html)                                                                                                                       |
+| [Geo-partitioned replicas](../v20.2/topology-geo-partitioned-replicas.html)        | [`REGIONAL BY ROW` tables](regional-tables.html#regional-by-row-tables) with [`ZONE` survival goals](multiregion-overview.html#surviving-zone-failures)     |
```
Comment (rmloveland) [inline _includes/v21.2/sql/replication-zone-patterns-to-multiregion-sql-mapping.md:4]: Guessing this is for the same reasons as above, and explanation will be the same

File: _redirects [hunk 4]
```diff
@@ -207,11 +207,14 @@ layout: null
...
-/docs/v21.1/topology-geo-partitioned-replicas.html /docs/v21.1/regional-tables.html 301
+/docs/v21.:version/topology-geo-partitioned-replicas.html /docs/v21.:version/regional-tables.html 301
+/docs/v22.:version/topology-geo-partitioned-replicas.html /docs/v22.:version/regional-tables.html 301
 /docs/stable/topology-geo-partitioned-replicas.html /docs/stable/regional-tables.html 301
-/docs/v21.1/topology-geo-partitioned-leaseholders.html /docs/v21.1/regional-tables.html 301
+/docs/v21.:version/topology-geo-partitioned-leaseholders.html /docs/v21.:version/regional-tables.html 301
+/docs/v22.:version/topology-geo-partitioned-leaseholders.html /docs/v22.:version/regional-tables.html 301
 /docs/stable/topology-geo-partitioned-leaseholders.html /docs/stable/regional-tables.html 301
-/docs/v21.1/topology-duplicate-indexes.html /docs/v21.1/global-tables.html 301
+/docs/v21.:version/topology-duplicate-indexes.html /docs/v21.:version/global-tables.html 301
```
Comment (rmloveland) [inline _redirects:216]: ah I guess the way we do redirects changed to use this file
I'm sure you mentioned something about it that I half heard / understood

File: v21.1/topology-duplicate-indexes.md [hunk 5]
```diff
@@ -1,683 +0,0 @@
----
```
Comment (rmloveland) [inline v21.1/topology-duplicate-indexes.md:1]: I don't understand the deletion, is it because we are 301 permanently redirecting away from this?

**Review by rmloveland — APPROVED** [review]
LGTM, I had some not very smart questions about the whole redirects setup but that was about it

For context there was a bit of a kerfuffle when we removed some of the legacy zone config-based geopartitioning docs a couple releases back and pointed to the new multi-region syntax, hence why I'm asking about them.  Were these pages already 301 redirecting anyway?  Or are we doing that here?

**Comment by rmloveland** [comment]
> For context there was a bit of a kerfuffle when we removed some of the legacy zone config-based geopartitioning docs a couple releases back and pointed to the new multi-region syntax, hence why I'm asking about them. Were these pages already 301 redirecting anyway? Or are we doing that here?

Ah ok, I just reviewed the Google doc where we discussed this context already.  All good on my end in that case.  If anyone yells about those pages getting removed (increasingly unlikely) I'll figure something out then.  It probably won't be necessary at this point.

File: _includes/v21.1/sql/movr-statements-geo-partitioned-replicas.md (hunk 1 above)
Comment (nickvigilante) [reply _includes/v21.1/sql/movr-statements-geo-partitioned-replicas.md:5]: Yes it is. I'm not sure why this redirect isn't working, but `/stable` is.

I'm also now discovering that the redirect just straight up does not work and produces a 404 if navigated to directly, so I think I need to tweak my redirect logic.

File: _includes/v21.2/sql/replication-zone-patterns-to-multiregion-sql-mapping.md (hunk 3 above)
Comment (nickvigilante) [reply _includes/v21.2/sql/replication-zone-patterns-to-multiregion-sql-mapping.md:4]: Yep, same explanation.

File: _includes/v21.2/sql/movr-statements-geo-partitioned-replicas.md (hunk 2 above)
Comment (nickvigilante) [reply _includes/v21.2/sql/movr-statements-geo-partitioned-replicas.md:5]: Yep, same explanation.

File: _redirects (hunk 4 above)
Comment (nickvigilante) [reply _redirects:216]: I made a note about this in Docs Weekly on Monday, but it seems like the syntax I used here doesn't work, so I gotta fix it.

File: v21.1/topology-duplicate-indexes.md (hunk 5 above)
Comment (nickvigilante) [reply v21.1/topology-duplicate-indexes.md:1]: That is correct.

File: _includes/v21.1/sql/movr-statements-geo-partitioned-replicas.md (hunk 1 above)
Comment (nickvigilante) [reply _includes/v21.1/sql/movr-statements-geo-partitioned-replicas.md:5]: Redirects fixed.

File: _redirects (hunk 4 above)
Comment (nickvigilante) [reply _redirects]: Redirects fixed.

**Comment by nickvigilante** [comment]
Closing in preference of https://github.com/cockroachdb/docs/pull/15498.

**Pull request closed without merging**